cumulusci/tasks/github.py

```diff
@@ -207,3 +210,51 @@ def _run_task(self):
                 )
 
                 self.logger.info('Merge conflict on branch {}: created pull request #{}'.format(branch.name, pull.number))
+
+
+class CommitApexDocs(BaseGithubTask):
+
+    task_options = {
```

Comment:
I'm not super familiar with how you run the apexdocs task. It looks like it doesn't have a default either.

I think it'd be great if the apexdocs task and this task shared a value there. This could be with a project config value that both tasks use as a default, or have the apexdocs task add the dir to return_values, and use that as the task option in a flow that generates *and* uploads.